### PR TITLE
Black all python files and add Github Actions check

### PR DESCRIPTION
**Pull request by sondreso**
**Approach**
Run black on all files inside `python`, remove black test in python tests and add Github Action check for black formatting.

### CODE REVIEW

**Comment by oyvindeide** [comment]
I agree with doing this, it is annoying to just have a subset being `blacked`. There is a way to avoid it cluttering blame: https://github.com/psf/black#migrating-your-code-style-without-ruining-git-blame

**Comment by jepebe** [comment]
In principal I agree, however, I know that some of these files would probably look much better with some human love before reformatting with black. Since it is 306 files I think maybe leave it as an exercise for the next person that enters the file?

**Comment by markusdregi** [comment]
> I agree with doing this, it is annoying to just have a subset being `blacked`. There is a way to avoid it cluttering blame: https://github.com/psf/black#migrating-your-code-style-without-ruining-git-blame

I agree, besides that that does not really help you when navigating the code on GitHub :)

**Comment by markusdregi** [comment]
I used to think that this was completely unheard of, as it will take a toll on the git history. However, I think it will never be completed unless we just do it 🤷 

I do agree with Jean-Paul on some manual aiding of the formatting though. Would it be worth it to do it module by module and add some manual labor to verify that the formatting is not completely crazy?

**Comment by sondreso** [comment]
> I think it will never be completed unless we just do it

This was my realization as well! 

> Would it be worth it to do it module by module and add some manual labor to verify that the formatting is not completely crazy?

I don't really see how we are going to do this. Because unless I'm missing something, then `better formatting -> refactoring`. And since not all of these files are in parts of the code that is currently prioritized in the ERT project, and think the best way is just ripping of the band-aid fast.

**Comment by markusdregi** [comment]
Rebase, split it into two commits and I'm inclined to approve this ;)

**Comment by sondreso** [comment]
Done! If this is merged I will add a second PR with the suggestions from here: https://github.com/psf/black#migrating-your-code-style-without-ruining-git-blame. But this will need to wait until we have the correct hash on master!

File: test_requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 pytest
 decorator
 pylint
-black==19.10b0
+black
```
Comment (markusdregi) [inline test_requirements.txt:4]: I'm being a bit pedantic now, but that is the best I can do when we are to apply a uncompromising code formatter 🤷 I don't think this, nor the removal of the formatting test should be touched in this commit. I would like it to purely be the result of applying black to the source code.

Maybe we can do something like this with the commits(?):
- Bump black to 19.10b0 (and update currently blacked lines)
- Apply black on all files (<- point is I want this to be a PR with no human edits)
- Move black test out of test_formatting and into CI

Comment (sondreso) [reply test_requirements.txt:4]: Done :D

**Review by markusdregi — APPROVED** [review]
Just what I had in mind 👏 Much easier to review!

Great initiative!

**Pull request merged**